termlink/__main__.py

```diff
@@ -106,6 +107,25 @@
     help="resource identifier for files"
 )
 
+parser_gsea = subparsers.add_parser(
+    "gsea",
+    help="Converts the 'Gene Set Enrichment Analysis Ontology'",
+    description="""
+    The Gene Set Enrichment Analysis Ontology (GSEA) project provides an ontology of genes grouped by a relational concept
+    """,
+    epilog="""
+    [1] Gene Set Enrichment Analysis Ontology. Retrieved May 8, 2019, from http://software.broadinstitute.org/gsea/msigdb/collections.jsp
+    """
+)
+
+parser_gsea.add_argument(
+    "uri",
+    metavar="URI",
+    help="resource identifier for files"
+)
+
+parser_gsea.set_defaults(execute=GSEACommand.execute)
+
 parser_rxnorm.set_defaults(execute=RxNormCommand.execute)
```

Comment:
Move this above the GSEA code block to keep it in the same code location as the RxNorm code.